ui/src/app/shared/shared.module.ts

```diff
@@ -110,7 +111,8 @@ export function SubnetmaskValidatorMessage(err, field: FormlyFieldConfig) {
     FormlyCheckBoxHyperlinkWrapperComponent,
     FormlyWrapperDefaultValueWithCasesComponent,
     FormlyFieldModalComponent,
-    PanelWrapperComponent
+    PanelWrapperComponent,
+    SkeletonWrapperComponent
```

Comment:
i think it needs to be defined in the formlyModule.forRoot section as well

Reply:
we are not extending formlyfields or using as a wrapper. only using formly-form in template. do we need to add it here?